### ISSUE TITLE
mapping from values.yaml does not work using camelcase or upper case characters

### ISSUE THREAD

**Issue by mmueller90** [bitgrip/cattlectl#5]
currently, I can't use upper case characters in keys in the values.yaml When trying to use them, I'm getting the error that it can't find the referenced property in the project.yaml
Example:
Values.yaml:
`apiserver_healthCheck_enabled: false`

project.yaml app part:
```
  - name: service-catalog
    catalog: service-catalog
    chart: catalog
    version: 0.1.39
    namespace: kube-system
    answers:
      apiserver.healthcheck.enabled: "{{ .apiserver_healthCheck_enabled}}"
```
```
>_ cattlectl apply -v1
2019/01/31 14:18:07 template: project:48:41: executing "project" at <.apiserver_healthChe...>: map has no entry for key "apiserver_healthCheck_enabled"
```
it would be nice to use camel case writing instead using for example "_" as seperation

**Comment by wtschreiter** [comment]
If we would only use a map unmarshaled from the values.yaml this would not be an issue.
We want to be able to overwrite all values in the values.yaml with environment variables.
That's why we'll not change this behaviour but check the values.yaml for this scenario and produce a proper error message.

We will also create a proper documentation about support restrictions to the values.yaml

**Comment by wtschreiter** [comment]
Closed by #9
